.github/PULL_REQUEST_TEMPLATE

```diff
@@ -6,7 +6,7 @@ Thanks for sending a pull request!  Here are some tips for you:
   3. Be sure to keep the PR description is updated to reflect all changes.
   4. Please write your PR title to summarize what this PR proposes.
   5. If this PR involves UI changes, please provide a screen capture (before/after) for a faster review.
-  6. Remember to add 'Fixes #ISSUE_NUMBER' to get the issue properly closed when the PR is merged.
+  6. Remember to add 'Fixes #ISSUE_NUMBER' to the end of your message body to get the issue properly closed when the PR is merged.
```

Comment:
should we add something like 
```suggestion
  6. Remember to add 'Fixes #ISSUE_NUMBER' (or any [valid keyword](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)) to the end of your message body to get the issue properly closed when the PR is merged.
```
so folks know it doesn't have to be `Fixes` in case they prefer others?